recipes-rubygems/rubygems-foo-bar_1.2.3.bb

```diff
@@ -0,0 +1,34 @@
+# SPDX-License-Identifier: GPL-2.0-only
```

Comment:
potential issue: Please add an 'SPDX-License-Identifier: MIT' comment to the file